include/solvers/navier_stokes_scratch_data.h

```diff
@@ -733,19 +733,48 @@ class NavierStokesScratchData
           cell_void_fraction[j] = cell_void_fraction_bulk;
       }
 
-
     if (particle_number != 0)
       {
         average_particle_velocity = average_particle_velocity / particle_number;
       }
 
-    // Vectors for interpolation of pressure gradients and velocity laplacians
-    // at particle positions
+    // Relative velocity and particle Reynolds
+    fluid_particle_relative_velocity_at_particle_location.resize(
+      particle_number + 1);
+    Re_particle.resize(particle_number + 1);
+    particle_number = 0;
+    for (auto &particle : pic)
```

Comment:
Wouldn't it be better to add these tasks to the same previous particle loop instead of looping over all particles again?